src/schemas/subject.rs

```diff
@@ -0,0 +1,42 @@
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Subject {
+    /// "The subject's name that is displayed when processing this certificate"
+    pub display_name: String,
+    /// "Contact information of the subject"
+    pub contact: Contact,
+    /// "The legal entity of the subject"
+    pub legal_entity: Option<LegalEntity>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct Contact {
+    /// "Contact email"
+    pub email: String,
+    #[serde(flatten)]
+    pub other: HashMap<String, Value>,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct LegalEntity {
+    /// "Name of the legal entity"
+    pub name: Option<String>,
+    /// "Official address of the legal entity"
+    pub address: Option<Address>,
+    #[serde(flatten)]
+    pub other: HashMap<String, Value>,
```

Comment:
Placeholder for additional fields.

Reply:
We do not defined LegalEntity as we are not processing that information in any kind of manner. This data will live in `Subject.additional_properties` with other additional information.